Source/DafnyCore/Backends/SinglePassCodeGenerator/SinglePassCodeGenerator.cs

```diff
@@ -1529,15 +1529,28 @@ public void Compile(Program program, ConcreteSyntaxTree wrx) {
       EmitFooter(program, wrx);
     }
 
-    protected (bool classIsExtern, bool included) GetIsExternAndIncluded(ClassLikeDecl cl) {
+
+    public bool IsImported(IAttributeBearingDeclaration me) {
```

Comment:
Would prefer `IsExternallyImported` since you made the other change, but could be changed in a future PR